src/components/Calendar.jsx

```diff
@@ -13,8 +13,15 @@ import {
 } from '@mdi/js';
 import Icon from '@mdi/react';
 import parse from 'html-react-parser';
-import { createRef, useCallback, useEffect, useMemo, useState } from 'react';
-
+import React, {
+  createRef,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import events from '../../dist/events.json';
```

Comment:
Pulling the `event.json` at time of build, so updated the PR build.

Reply:
Interestingly, the performance boost is so big already from the static change alone, that we do not necessarily need my suggested change at all.